test/views/updateRegisteredDetails/summary/BusinessAddressSummarySpec.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.updateRegisteredDetails.summary
+
+import base.SpecBase
+import models.backend.UkAddress
+import views.summary.updateRegisteredDetails.BusinessAddressSummary
+
+class BusinessAddressSummarySpec extends SpecBase {
+  lazy val businessAddress: UkAddress = UkAddress(List("33 Rhes Priordy", "East London"), "E73 2RP")
+
+  "row" - {
```

Comment:
"BusinessAddressSummarySpec" ?

Reply:
Yep - officially the summary file is what is creating the ability for the view.  It reduces it needing to be tested by the time the CYA page comes